pkgs/applications/networking/instant-messengers/qq/default.nix

```diff
@@ -74,8 +75,7 @@ stdenv.mkDerivation {
       --replace "/opt/QQ/qq" "$out/bin/qq" \
       --replace "/usr/share" "$out/share"
     makeWrapper $out/opt/QQ/qq $out/bin/qq \
-      --argv0 "qq" \
-      --prefix GTK_IM_MODULE_FILE : "/run/current-system/sw/etc/gtk-3.0/immodules.cache"
+      --argv0 "qq"
```

Comment:
Can we remove the `makeWrapper` and just link it now?

Reply:
Ok, sure